compiler/codegen/mli.cpp

```diff
@@ -414,7 +414,7 @@ std::string MLIContext::genMarshalBodyChplBytesWrapper(Type* t, bool push) {
   std::string gen;
 
   // TODO: Should do something with the error code.
-  gen += push ? "chpl_mli_push" : "chpl_mli_pull";
+  gen += push ? mliPushFunctionName : mliPullFunctionName;
   gen += "_byte_buffer(skt,&";
```

Comment:
Note that this one had tried to share the `_` so you'll want to remove that from the start of this line

Reply:
Scratch that, I misread.  The globals don't have the `_`, but the other uses of the name were relying on them having it.